### DISCUSSION TITLE
RawText

### DISCUSSION THREAD

**Discussion by neoacevedo** [schultek/jaspr#333] [General]
What's the purpose for RawText? According with https://docs.page/schultek/jaspr/components/raw_text, it should render unescaped HTML but the reality is different. If I use the example code, for instance, inside a div, the content will be cleaned leaving the parent div without content, so, if I like to create a raw svg the problem will be the same: a svg tag without any content due to RawText is cleaning the content.

**Comment by schultek** [comment]
Can you show an example?

**Comment by neoacevedo** [comment]
I'm running the project in static mode (may be the reason?), and I want to add a raw svg image:

```dart
raw(
    '<svg xmlns="http://www.w3.org/2000/svg" id="Layer_3" data-name="Layer 3" viewBox="0 0 64 64" width="512" height="512">' 
    '    <path d="M62.707,31.293l-30-30a1,1,0,0,0-1.414,0l-30,30A1,1,0,0,0,2,33H7V57H6a1,1,0,0,0-1,1v4a1,1,0,0,0,1,1H58a1,1,0,0,0,1-1V58a1,1,0,0,0-1-1H57V33h5a1,1,0,0,0,.707-1.707ZM57,61H7V59H57ZM9,57V32.414l23-23,23,23V57ZM56.414,31,32.707,7.293a1,1,0,0,0-1.414,0L7.586,31H4.414L32,3.414,59.586,31Z"/>' 
    '    <path d="M41,39H36V36a1,1,0,0,0-1-1H29a1,1,0,0,0-1,1v3H23a3,3,0,0,0-3,3v4a1,1,0,0,0,1,1h1v7a1,1,0,0,0,1,1H41a1,1,0,0,0,1-1V47h1a1,1,0,0,0,1-1V42A3,3,0,0,0,41,39ZM30,37h4v2H30ZM40,53H24V47h5v3a1,1,0,0,0,1,1h4a1,1,0,0,0,1-1V47h5Zm-9-6h2v2H31Zm11-2H22V42a1,1,0,0,1,1-1H41a1,1,0,0,1,1,1Z"/>' 
    '    <path d="M46,31H42.422a1,1,0,0,1,0-2H46a1,1,0,0,0,.916-1.4A5.992,5.992,0,0,0,36.231,27h-8.04a5.992,5.992,0,0,0-10.685.6,1,1,0,0,0,.916,1.4H22a1,1,0,0,1,0,2H18.422a1,1,0,0,0-.916,1.4,5.992,5.992,0,0,0,10.685.6h8.04a5.992,5.992,0,0,0,10.685-.6A1,1,0,0,0,46,31Zm-4.578,3a4,4,0,0,1-3.662-2.4,1,1,0,0,0-.916-.6H27.578a1,1,0,0,0-.916.6,3.991,3.991,0,0,1-6.3,1.4H22a3,3,0,0,0,0-6H20.359a3.991,3.991,0,0,1,6.3,1.4,1,1,0,0,0,.916.6h9.266a1,1,0,0,0,.916-.6,3.991,3.991,0,0,1,6.3-1.4h-1.64a3,3,0,0,0,0,6h1.64A3.988,3.988,0,0,1,41.422,34Z"/>'
    '</svg>'
);
```

But when I run it, the raw disapears.

**Reply by neoacevedo** [reply]
Another thing, I'm clueless if is a mistake from my side, it should be a bug to post in the issues, but if I try to use Google Maps or Leaflet only from javascript, setting the scripts as defer, the maps loads at starting but some milliseconds later the map content inside the div that should contain the map gets empty. It's a similar performance with the raw html.

**Reply by schultek** [reply]
The RawText issue should be fixed in the newest version, this was indeed a bug.

For the other thing please open a separate issue and put a code snippet that shows this.
